IFC4x3/Properties/t/TypeOfSupportingSystem_3WKZxWJOXCiOu_tvXdPAUd/DocProperty.xml

```diff
@@ -1,5 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TypeOfSupportingSystem_3WKZxWJOXCiOu_tvXdPAUd" Name="TypeOfSupportingSystem" UniqueId="e0523ee0-4d88-4cb1-8e3e-df986764a7a7" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
-	<Enumeration xsi:type="DocPropertyEnumeration" xsi:nil="true" href="PEnum_SupportingSystemType" />
-</DocProperty>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TypeOfSupportingSystem_3WKZxWJOXCiOu_tvXdPAUd" Name="TypeOfSupportingSystem" UniqueId="e0523ee0-4d88-4cb1-8e3e-df986764a7a7" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel" />
```

Comment:
This should not be a p_enumeratedvalue if the enumeration was removed.

Reply:
This was an error it is a property from another PR set.
reverted